packages/eligibility/src/Eligibility.js

```diff
@@ -19,15 +21,56 @@ function Eligibility() {
   const [wrongQuestions, setWrongQuestions] = useState([]);
   const [maybeQuestions, setMaybeQuestions] = useState([]);
   const [resultClick, setResultClick] = useState(null);
+  const [cookies, setCookie] = useCookies(["uuid"]);
+  const [values, setValues] = useState({});
 
   React.useEffect(() => {
     document.addEventListener('keydown', handleKeys);
 
+    // Initialize cookie if not present
+    const userId = uuidv4();
+    if (!cookies.uuid) {
+      setCookie("uuid", userId, {path: "/", maxAge: 2592000}); // maxAge: 30 days
+    } 
+
     return () => {
       document.removeEventListener('keydown', handleKeys);
     };
   }, []);
 
+  const debounce = (func, wait) => {
+    let timeout;
+    return function (...args) {
+      const context = this;
+      if (timeout) clearTimeout(timeout);
+      timeout = setTimeout(() => {
+        timeout = null;
+        func.apply(context, args);
+      }, wait);
+    };
+  };
+
+  const debouncedSave = useCallback(
+    debounce((vals) => saveToDb(vals), 1000),
+    [cookies.uuid]
+  );
+
+  async function saveToDb(vals) {
+    if (cookies.uuid) {
+      console.log("Enters saveToDb");
```

Comment:
Please remove `console.log()` statements from production code.

Reply:
Resolved in commit 0edde98.